src/App.js

```diff
@@ -15,7 +15,7 @@ const App = () => {
     <IntlProvider locale="en">
       <React.Fragment>
         <NotificationsPortal />
-        <Router basename="/">
+        <Router basename="/cloud-connect">
```

Comment:
I assume there is no beta env?

Reply:
Not at the moment. Since this app is deployed directly (without the chroming service) we will have to discuss the `beta` environment later down the line.